pilot/pkg/config/kube/crdclient/client.go

```diff
@@ -174,10 +174,19 @@ func NewForSchemas(ctx context.Context, client kube.Client, revision, domainSuff
 	for _, s := range schemas.All() {
 		// From the spec: "Its name MUST be in the format <.spec.name>.<.spec.group>."
 		name := fmt.Sprintf("%s.%s", s.Resource().Plural(), s.Resource().Group())
-		if _, f := known[name]; f {
+		crd := true
+		switch s.Resource().Group() {
+		case gvk.Pod.Group, gvk.Deployment.Group, gvk.MutatingWebhookConfiguration.Group, gvk.CustomResourceDefinition.Group:
```

Comment:
This is the hack point

Reply:
Don't we have other k8s native type? Like Node, ValidatingWebhookConfiguration?

I wonder we can cache all the k8s api resources, and if one resource kind  is in,  it is native.